.github/workflows/spriggit-out.yml

```diff
@@ -0,0 +1,54 @@
+name: spriggit-out
+on:
+  schedule:
+    - cron: '0 9 * * *'
+  workflow_dispatch:
+jobs:
+  publish:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Fetch spriggit
+        shell: bash
+        run: gh release download -O .github/spriggit.zip -p 'SpriggitLinuxCLI.zip' -R Mutagen-Modding/Spriggit ${{ inputs.spriggit_tool_version }}
+        env:
+          GH_TOKEN: ${{ github.token }}
+          
+      - name: Extract some files
+        shell: bash
+        run: 7z x .github/spriggit.zip -o.github/spriggit
```

Comment:
doesn't this need space between `-o` and `.github/spriggit`?

Reply:
I thought so too.  If you look at any examples online though, they all omit the space, and just gloss over it, lol.